llvm/unittests/ADT/EquivalenceClassesTest.cpp

```diff
@@ -77,7 +77,6 @@ TEST(EquivalenceClassesTest, MembersIterator) {
 
   EquivalenceClasses<int>::iterator I = EC.findValue(EC.getLeaderValue(1));
   EXPECT_THAT(EC.members(I), testing::ElementsAre(5, 1, 2));
-  EXPECT_EQ(EC.members(EC.end()).begin(), EC.member_end());
 }
```

Comment:
For example, you can leave a `TODO` comment here that mentions we should test this.

Reply:
Thanks, will do on second re-land.